### ISSUE TITLE
will you release the training dataset and the training code?

### ISSUE THREAD

**Issue by Ir1d** [wangruixing/DeepUPE#21]
also, the test set is not shared publicly in this link: https://drive.google.com/file/d/1HZnNgptNxjKJAhekz2K5yh0mW0yKIws2/view?usp=sharing

![image](https://user-images.githubusercontent.com/10709657/61101716-63a3a300-a49d-11e9-958f-1897f3fb0109.png)
